pkgs/development/libraries/exiv2/default.nix

```diff
@@ -103,6 +103,15 @@ stdenv.mkDerivation rec {
     remove-references-to -t ${stdenv.cc.cc} $lib/lib/*.so.*.*.* $out/bin/exiv2 $static/lib/*.a
   '';
 
+  postFixup = ''
+    substituteInPlace "$dev"/lib/cmake/exiv2/exiv2Config.cmake --replace \
+      "set(_IMPORT_PREFIX \"$out\")" \
+      "set(_IMPORT_PREFIX \"$static\")"
```

Comment:
I would be worried about the non-static library. Maybe we need a second `set(_IMPORT_PREFIX "$out")` before `add_library(exiv2lib SHARED IMPORTED)`?

Reply:
I was also worried.  But then I noticed that the shared library is not in `$out` but in `$lib`.

Reply:
Oh, now I see that just changed a few days ago, so it might cause some breakage that hasn't been discovered yet.

Reply:
If you think you understand how this cmake file works, I'm certainly not against.  We'd probably need a different approach for that, maybe a diff, but that's just details.